doc/release-process.md

```diff
@@ -42,6 +42,8 @@ Check out the source code in the following directory hierarchy.
 
 ### Bitcoin maintainers/release engineers, suggestion for writing release notes
 
+Open a release notes draft for collaborative editing at https://github.com/bitcoin-core/bitcoin-devwiki/wiki
```

Comment:
Good to mention.
Though if you do, I think it needs to also mention merging this wiki back into the branch just before `-final`